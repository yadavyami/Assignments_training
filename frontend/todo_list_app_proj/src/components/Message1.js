import React from 'react'

const Message1 = ({text1, message}) => {
  return (
    <div>
      { text1 }
      { message }
    </div>
  )
}

export default Message1
