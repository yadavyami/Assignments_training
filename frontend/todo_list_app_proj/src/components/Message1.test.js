import React from 'react'
import renderer from 'react-test-renderer'
import Message1 from './Message1'

test("Message snapshot with 2 parameters", () => {
    const component = renderer.create(<Message1 text1="Hello, Aditya!" message="Welcome" />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})