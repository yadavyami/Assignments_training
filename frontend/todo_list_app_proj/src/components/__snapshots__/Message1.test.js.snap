// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message snapshot with 2 parameters 1`] = `
<div>
  Hello, Aditya!
  Welcome
</div>
`;
